src/ray/core_worker/core_worker.cc

```diff
@@ -5,13 +5,13 @@
 
 namespace ray {
 
-CoreWorker::CoreWorker(
-    const WorkerType worker_type, const Language language,
-    const std::string &store_socket, const std::string &raylet_socket,
-    const JobID &job_id, const gcs::GcsClientOptions &gcs_options,
-    const std::string &log_dir, const std::string &node_ip_address,
-    const CoreWorkerTaskExecutionInterface::TaskExecutor &execution_callback,
-    bool use_memory_store)
+CoreWorker::CoreWorker(const WorkerType worker_type, const Language language,
+                       const std::string &store_socket, const std::string &raylet_socket,
+                       const JobID &job_id, const gcs::GcsClientOptions &gcs_options,
+                       const std::string &log_dir, const std::string &node_ip_address,
+                       const CoreWorkerTaskExecutionInterface::TaskExecutionCallback
+                           &task_execution_callback,
```

Comment:
Maybe call this `task_execution_handler`? I kept thinking this was to be called once a task finished execution, but that's not correct, right?